### ISSUE TITLE
Formatting according to rule 3.6.2 is inconsistent sometimes

### ISSUE THREAD

**Issue by petertrr** [saveourtool/diktat#949]
## Describe the bug
If there are more than threshold consecutive dot-qualified function calls and some of them are multiline lambdas, we don't fix formatting of these `< threshold` calls according to functional style:
```kotlin
val x = foo.bar {
}.bar {}.bar().baz {
        
    }
```
should be fixed as
```kotlin
val x = foo.bar {
}
    .bar {}
    .bar()
    .baz {
        
    }
```

## Environment information
* diktat version: 1.0.0-rc.1

**Comment by nulls** [comment]
tested on demo project: code formatted to
```kotlin
val x = foo.bar {
    }
    .bar {}
    .bar()
    .baz {
        
        }
```

It looks as expected result
